src/ejabberd_sm.erl

```diff
@@ -312,16 +315,16 @@ get_session_pid(JID) ->
 
 -spec get_unique_sessions_number() -> integer().
 get_unique_sessions_number() ->
+    InternalDB = get_internal_db(),
     try
         C = ejabberd_sm_backend:unique_count(),
-        mongoose_metrics:update(global, ?UNIQUE_COUNT_CACHE, C),
+        set_cached_unique_count(C, InternalDB),
```

Comment:
At the grooming we agreed that we could just remove this cache entirely, what's the reason for the change? Is it used anywhere?